.babelrc

```diff
@@ -1,11 +1,14 @@
 {
-	"presets": [
-		[
-			"@babel/preset-env", {
-				"targets": {
-					"node": "8"
-				}
-			}
-		]
-	]
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "debug": true,
```

Comment:
Is this `debug` statement something that we really want? I'd think that we wouldn't want that enabled

Reply:
```suggestion
```

sorry no, left over from my debugging